autorest/autorest_test.go

```diff
@@ -0,0 +1,79 @@
+package autorest
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2017-06-01/storage"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+var accountName = "testname01"
```

Comment:
As much as possible, I'd like to refrain from having global variables. Could this at least be a `const`?